tests/unit/test_log_parser.py

```diff
@@ -52,11 +51,9 @@ def test_old_ldlogger(self):
         # option_parser, so here we aim for a non-failing stalemate of the
         # define being considered a file and ignored, for now.
 
-        build_action = log_parser.parse_log(logfile, ParseLogOptions())[0]
-        results = option_parser.parse_options(build_action.original_command)
+        build_action = log_parser.parse_log(load_json_or_empty(logfile))[0]
 
-        self.assertEqual(' '.join(results.files),
-                         r'"-DVARIABLE="some value"" /tmp/a.cpp')
+        self.assertEqual(build_action.source, r'/tmp/a.cpp')
```

Comment:
Do we still have a test for compiler flags like this `"-DVARIABLE="some value"" /tmp/a.cpp`

Reply:
Note that, if we want to support windows in the future we might want to use a Python standard library function to get a temporary file rather than hardcoding a name. This would also reduce the chance of colliding file names (even when multiple test instances are running in parallel on the same machine).